uberfire-workbench/uberfire-workbench-client/src/main/java/org/uberfire/client/mvp/WorkbenchClientEditorActivity.java

```diff
@@ -26,11 +26,15 @@
 public interface WorkbenchClientEditorActivity extends WorkbenchActivity {
 
     /**
-     * Set the editor content
+     *  
+     *  Set the editor content
+     *  
+     * @param path
+     * Content Path 
```

Comment:
Improved the comment to say that it is the relative path - in fact, full path does not make sense in GITHUB channel, it would make sense in VSCODE and in the future DESKTOP channel (full path of a file in the disk), so relative paths should be always the choice.